doc-content/enterprise-only/openshift/proc-dashbuilder-standalone.adoc

```diff
@@ -0,0 +1,77 @@
+[id="proc-dashbuilder-standalone_{context}"]
+= Installing Dashbuilder Standalone using Operator
+
+Use Operator to install Dashbuilder Standalone separately from other services.
+
+.Procedure
+
+. On the *Installation* page, enter `standalone-dashbuilder` in the *Application name* field.
```

Comment:
user can choose anything here, why "enter ...." not just fill the field with the desired content?

Reply:
We use "enter" to describe inputting information. It is taken from the IBM Style Guide. Do you think it is a good idea to remove the specific example text?

s/On the *Installation* page, enter `standalone-dashbuilder` in the *Application name* field./On the *Installation* page, enter a name for the application in the *Application name* field.